ipmi-config/ipmi-config.h

```diff
@@ -276,6 +276,14 @@ struct ipmi_config_state_data
   int enable_user_after_password_len;
   ipmi_config_enable_user_after_password_t *enable_user_after_password;
 
+  /* lamont: caching to make ipv6 go faster */
+  int ipv6_ipv4_support_initialized;
+  uint8_t ipv6_ipv4_support_channel_number;
+  uint8_t ipv6_ipv4_support_supports_ipv6;
```

Comment:
After reading the code in a more-awake state, I see what you mean.  I've dropped ipv6_ipv4_support_supports_ipv6 from the code.